src/components/cesium_map/cesium_UI.js

```diff
@@ -27,8 +28,10 @@ Cesium.Ion.defaultAccessToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOi
 let lat = -17.451466233002286;
 let long = -149.8169236266867;
```

Comment:
Are these actually used anywhere? It's confusing to have global vars with the same name as vars used as method parameters (e.g. the showCoordinates method)